src/main/kotlin/Stores/ChannelStore.kt

```diff
@@ -0,0 +1,41 @@
+package Stores
+
+import DatabaseFactory
+import io.ktor.http.Parameters
+import models.Channel
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.select
+import org.joda.time.DateTime
+import java.util.*
+
+object ChannelStore {
+    suspend fun createStore(params: Parameters): String? {
```

Comment:
Let's not call the `createStore` since that's inaccurate. We are creating a channel, not a store. maybe `createChannel`?